server/www/templates/posts/detail.html

```diff
@@ -61,7 +61,11 @@ <h1 class="page-title">
                        <p>
                           <strong class="icon-left fa-circle">
                               {{form_name || "Unstructured"}}
-                          </strong> posted at {{post.updated || post.created | date:'shortTime'}}<strong> {{post.updated || post.created | date:'mediumDate'}}</strong> <br> <span ng-if="post.user.realname != null"><translate>post.by</translate> <a href="/settings/users/{{post.user.id}}">{{post.user.realname}}</a></span>
+                          </strong> posted at {{post.updated || post.created | date:'shortTime'}}<strong> {{post.updated || post.created | date:'mediumDate'}}</strong> <br>
```

Comment:
I suspect this should all be a single translation string ie. "posted at {{time}} by {{source}} etc" .. but not sure how to handle the HTML in that.